internal/format/asciidoc_document.go

```diff
@@ -45,6 +45,15 @@ const (
 	{{ end -}}
 	`
 
+	asciidocDocumentFooterTpl = `
+	{{- if .Settings.ShowFooter -}}
+		{{- with .Module.Footer -}}
+			{{ sanitizeHeader . }}
```

Comment:
Nit: I'm a bit leaning towards renaming this function, maybe `sanitizeSection`? If you have any suggestion please let me know, otherwise we can leave this alone for time being as we might refactor templating a bit and this can be addressed then.

Reply:
Sounds good. I think I intended to come back to this once I had working tests then forgot!